apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -138,8 +139,20 @@ export const genDefaultQuery = (table: LogsTableName, filters: Filters) => {
   const where = genWhereStatement(table, filters)
   const joins = genCrossJoinUnnests(table)
   const orderBy = 'order by timestamp desc'
+
   switch (table) {
     case 'edge_logs':
+      if (IS_PLATFORM === false) {
+        return `
+-- local dev edge_logs query
+select id, edge_logs.timestamp, event_message, metadata.event_type, metadata.function_id, metadata.level 
```

Comment:
why function_id and event_type?

Reply:
was using another function as reference and i probably copypasted it 😅 nice catch thx